opm/common/OpmLog/LogBackend.cpp

```diff
@@ -33,23 +33,49 @@ namespace Opm {
 
     void LogBackend::configureDecoration(std::shared_ptr<MessageFormatterInterface> formatter)
     {
-        formatter_ = formatter;
+        m_formatter = formatter;
+    }
+
+    void LogBackend::configureMessageLimiter(std::shared_ptr<MessageLimiter> limiter)
```

Comment:
I agree and think we should change configureDecoration to setMessageFormatter as well.